### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by harry-nicolainve**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,6 +6,6 @@
   <title>Welcome to Potionfy</title>
 </head>
 <body>
-  
+  <h1>Welcome to Potionfy! Enjoy your stay.</h1>
```
Comment (hermione-nicolainve) [inline index.html:9]: Could you remove the `Enjoy your stay.` string and make the `Welcome to Potionfy!` string red?

**Comment by harry-nicolainve** [comment]
Sure, can do!

**Review by hermione-nicolainve — CHANGES_REQUESTED** [review]
Please, change the code as per my comments on the review.

**Review by hermione-nicolainve — APPROVED** [review]
Well done Harry!

**Pull request merged**